tests/eclient/image/get_eve_ip.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
+
+EVE_IP=$(curl -s "http://169.254.169.254/eve/v1/network.json" | jq -r '."external-ipv4"')
```

Comment:
Looks it's a strange magic number. Maybe even wrong?

Reply:
https://serverfault.com/questions/427018/what-is-this-ip-address-169-254-169-254

Reply:
EVE is the meta-data distributor in this case. Please let me know if it still looks wrong.

Reply:
@mydatascience The eve-specific URL is documented in https://github.com/lf-edge/eve/blob/master/docs/ECO-METADATA.md The generic use of http://169.254.169.254/ is part of cloud-init.

Reply:
got it